app/assets/stylesheets/layouts/starry.scss

```diff
@@ -1,85 +1,217 @@
 @import 'variables';
+
+/* - header */
+$bg_color_header_top: transparent; // top header strip for logo
+$bg_color_header_middle: #05050F; // middle header strip with user items
+$bg_color_header_bottom: #484357; // bottom header strip with navigation
+
 #header {
-  background-color: #484357;
+  background-color: $bg_color_header_bottom;
   background-image: url(image-path("layouts/starry.jpg"));
   background-position: right top;
 }
-#logo, #nav-bottom { background-color: transparent !important; }
-#user-info, #nav-top { background-color: #05050F; background-color: rgba(5, 5, 15, 0.9); }
+#logo { background-color: $bg_color_header_top !important; }
+#user-info, #nav-top { background-color: $bg_color_header_middle; background-color: rgba($bg_color_header_middle, 0.9); }
 
-body { background-color: #6C697C; }
-#character-icon-selector, .time-loaded { color: #DADADA; }
+/* - links */
+$font_color_link: #666284; // link
+$font_color_link_visited: #9C9AA4;
 
-th, .content-header { background-color: #13132F; }
-.sub { color: #F3F3F3; }
-.sub, .post-navheader, .flash.breadcrumbs { background-color: #182152; }
-.form-table-ender { background-color: #13132F; color: #F3F3F3; }
-.selector-popup, .single-description, .subber { background-color: #484357; color: #DADADA; }
-th, .single-description, .content-header {
-  a { color: #B5AFCB; }
-  a:visited { color: #9C9AA4; }
+a { color: $font_color_link; }
+a:visited { color: #484357; } //TODO: inconsistency between this a:visited and others, ???? this was previous at the bottom of the documment, was it overriding?
+
+/* - flashes */
+// these are the notifications that appear when you submit a form or get a message
+$bg_color_flash_error: #9692A8; // error
+$bg_color_flash_success: #9692A8; // success
+$bg_color_flash_inbox: #9692A8; // new message // TODO: THESE SHOULD NOT ALL THE SAME
+$bg_color_flash_breadcrumbs: #182152; // breadcrumbs ("Continuities » Continuity » Section » Thread")
+$font_color_flash_bold: #F3F3F3;
+
+.flash.breadcrumbs {
+  background-color: $bg_color_flash_breadcrumbs;
+  &, a, a:visited { color: $font_color_link_visited; }
+  b { color: $font_color_flash_bold; }
 }
+.flash.success { background-color: $bg_color_flash_success; }
+.flash.error { background-color: $bg_color_flash_error; }
+.flash.inbox { background-color: $bg_color_flash_inbox; }
 
-$EVEN_COLOR: #E3E3E3;
-$ODD_COLOR: #D8D8D8;
+/* - main page */
+$bg_color_page: #6C697C; // overall page
+$font_color_page: #DADADA;
```

Comment:
There was a problem setting the page font color like this, I think because it affects flashes. This works if we check for all elements that can be root on the page and fix the colors if they give a background, but I previously determined that too much effort when I was trying to fix it, and it looks like this has caused some problems if you actually load pages in the relevant layout.

Reply:
Commented out that use of that variable